docs/cleanup/README.md

```diff
@@ -0,0 +1,112 @@
+# TVK Cleanup Plugin
+
+**tvk-cleanup** is a kubectl plugin which cleans up Triliovaultfor Kubernetes 
+application, Custom reources and CRDs in a Kubernetes cluster.
+
+## Pre-requisites:
+
+1. krew - kubectl-plugin manager. Install from [here](https://krew.sigs.k8s.io/docs/user-guide/setup/install/)
+2. kubectl - kubernetes command-line tool. Install from [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+**Supported OS and Architectures**:
+
+OS:
+- Linux
+- darwin
+
+Arch:
+- amd64
+- x86
+
+
+## TVK Cleanup
+
+This plugin cleans up all TVK Custom Resources, CRDs, and TVK application itself from all the namespace.
+It cleans up TVK installed as operator (on OCP platform) and as helm chart on upstream k8s (Rancher) cluster.
+
+Please note the following:
+- Ensure *kubectl* utility is present on system
+- Ensure *kubectl* is pointed to k8s cluster (i.e can access the remote target cluster)
+- Ensure *helm*  utility is present on system and pointed to the cluster
+  - If *helmVersion=~v3*, then *tiller* is not needed on cluster
```

Comment:
remove this and just write - `helmVersion=~v3` is needed `

Reply:
Done